jujugui/static/gui/src/app/components/deployment-flow/signup/signup.js

```diff
@@ -0,0 +1,144 @@
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('deployment-signup', function() {
+
+  juju.components.DeploymentSignup = React.createClass({
+    propTypes: {
+      changeState: React.PropTypes.func.isRequired,
+      exportEnvironmentFile: React.PropTypes.func.isRequired,
+      modelName: React.PropTypes.string.isRequired
+    },
+
+    /**
+      Handle navigating to the deployment flow.
+
+      @method _displayFlow
+    */
+    _displayFlow: function() {
+      this.props.changeState({
+        sectionC: {
+          component: 'deploy',
+          metadata: {
+            activeComponent: 'flow'
+          }
+        }
+      });
+    },
+
+    /**
+      Handle the email registration.
+
+      @method _handleSignup
+    */
+    _handleSignup: function() {
+      console.error('Not implemented.');
+    },
+
+    render: function() {
+      return (
+        <juju.components.DeploymentPanel
+          changeState={this.props.changeState}
+          title={this.props.modelName}>
+          <div className="deployment-signup">
+            <div className="six-col">
+              <h2>Install Juju to deploy locally</h2>
+              <p>
+                Doing so provides on LXD, which allows you to recreate the
+                production deployment environment on your own machine. This
+                minimises portability issues when deploying to a public cloud,
+                OpenStack or bare metal.
+              </p>
+              <p>To deploy locally:</p>
+              <ol>
+                <li>
+                  <juju.components.GenericButton
+                    action={this.props.exportEnvironmentFile}
+                    type="base"
+                    title="Download your model" />
+                  </li>
+                <li>
+                  <a href="https://jujucharms.com/docs/stable/getting-started"
+                    target="_blank">
+                    Install Juju
+                  </a>
+                </li>
+                <li>Add your model to deploy</li>
+              </ol>
+              <p>
+                Or continue to the&nbsp;
+                <juju.components.GenericButton
+                  action={this._displayFlow}
+                  type="inline-neutral"
+                  title="deployment demo of Juju" />
+              </p>
+            </div>
+            <div className="twelve-col">
+              <h2>Lorem ipsum dolor sit</h2>
```

Comment:
Thank you for just lorem ipsuming this so we could get past the block waiting on wording.